### DISCUSSION TITLE
web browser games

### DISCUSSION THREAD

**Discussion by gnuchanos** [qutebrowser/qutebrowser#7531] [Q&A]
for example ı try play games on itch.io ı press full screen and ı press d window close and browser become full screen ı can't exit why can't see games qutebrowser user not play game ?

**Comment by toofar** [comment]
I'm not 100% sure I'm reading your question right. But I'll try a couple of answers.

If you are saying you get stuck in fullscreen mode after closing the fullscreen tab with the game in it, then you should be able to put the browser window back to normal size by pressing F11.

If you are saying you should be able to press `d` in the game without closing it then you probably want to enter passthrough mode (bound to shift+v by default) before starting it.

**Reply by gnuchanos** [reply]
shift+v not work with games for example you can't press enter in game just go itch.io try play web game

**Reply by toofar** [reply]
Sorry, it is ctrl+v! I wrote the wrong thing. You should get a blue status bar when you are in passthrough mode, and not in fullscreen. You don't get any indication when you are in fullscreen mode, but remember it's shift+escape to get back to normal mode.

Works for me fine on https://zephyo.itch.io/an-average-day-at-the-cat-cafe

**Reply by gnuchanos** [reply]
why this browser can't see web games or don't have game mode ? no one play game in browser this community

**Reply by The-Compiler** [reply]
It's still very hard to tell what issue it is that you're having. What do you mean with "can't see web games"? What would such a "game mode" be?
